src/Microsoft.Diagnostics.TestHelpers/Microsoft.Diagnostics.TestHelpers.csproj

```diff
@@ -13,6 +13,7 @@
   </PropertyGroup>
   
   <ItemGroup>
+    <PackageReference Include="Microsoft.Windows.Compatibility" Version="3.1.0" />
```

Comment:
Why does the TestHelpers assembly need this reference? It doesn't have any "ps" command line options (it doesn't have any options).

Reply:
That was added by mistake. Will remove the dependency.